app/assets/v2/images/avatar/Accessories/Earring-0.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
We'll want to run `svgo` against all of the newly added SVGs before this gets merged.
Instructions here: https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#contributing-static-assets

Reply:
will do once confirmed all svgs are added